sonarqube-companion-rest/src/main/java/pl/consdata/ico/sqcompanion/members/MemberService.java

```diff
@@ -34,16 +38,34 @@ public void syncLocalMembers() {
                         .id(definition.getUuid())
                         .aliases(definition.getAliases())
                         .mail(definition.getMail())
+                        .remote(false)
                         .build()
                 )
                 .collect(Collectors.toList()));
+        appConfig.getMembers().getLocal().forEach(this::addDetachedEvents);
+        appConfig.getMembers().getLocal().forEach(this::addAttachedEvents);
+    }
 
-        appConfig.getMembers().getLocal().forEach(this::addLocalDetachedEvents);
-        appConfig.getMembers().getLocal().forEach(this::addLocalAttachedEvents);
-        log.info("< Synced local members");
+    private void syncRemoteMembers() {
+        log.info("> Sync remote members");
+        List<Member> remoteMembers = membersIntegrations.getMembers();
+        memberRepository.saveAll(remoteMembers
+                .stream().map(definition -> MemberEntryEntity.builder()
+                        .firstName(definition.getFirstName())
+                        .lastName(definition.getLastName())
+                        .id(definition.getUuid())
+                        .aliases(definition.getAliases())
+                        .mail(definition.getMail())
+                        .remote(definition.isRemote())
+                        .remoteType(definition.getRemoteType())
+                        .build()
+                )
+                .collect(Collectors.toList()));
+        remoteMembers.forEach(this::addDetachedEvents);
+        remoteMembers.forEach(this::addAttachedEvents);
     }
 
-    private void addLocalAttachedEvents(Member member) {
+    private void addAttachedEvents(Member member) {
```

Comment:
co to jest attached i detached?

Reply:
rename, mam nadzieję, że tera nazwa więcej mówi